### ISSUE TITLE
no provider for rendered2

### ISSUE THREAD

**Issue by JoaoPintoM** [MurhafSousli/ngx-disqus#28]
hello, 

thx for your work, but i got this error : 

```
ERROR Error: StaticInjectorError[Renderer2]: 
  StaticInjectorError[Renderer2]: 
    NullInjectorError: No provider for Renderer2!
    at _NullInjector.get (core.js:923)
```

do you have any idea from where it's from ?

**Comment by MurhafSousli** [comment]
Hi @JoaoPintoM I have no idea! I just updated `ng-packagr` and compiled the package again, Try with **v2.3.1** and let me know if it works! if it did not, add your usage code

**Comment by kristaps-ancans** [comment]
Same here. 
Error happens with Angular 5.0.1 using ngx-disqus 2.3.1

**Comment by MurhafSousli** [comment]
@kristaps-ancans @JoaoPintoM Fixed in **v2.3.4** here is a [stackblitz](https://stackblitz.com/edit/ngx-disqus)

**Comment by kristaps-ancans** [comment]
@MurhafSousli tested on Angular 5.0.1 - still the same.

**Comment by MurhafSousli** [comment]
@kristaps-ancans remove `node_modules` and reinstall again, have a look at the [stackblitz](https://stackblitz.com/edit/ngx-disqus) I posted above

**Comment by kristaps-ancans** [comment]
@MurhafSousli already removed node_modules, reinstalled. In stackblitz angular version is 5.0.0. Try with latest or at least 5.0.1.

**Comment by MurhafSousli** [comment]
@kristaps-ancans Please reproduce your issue in a stackblitz

**Comment by kristaps-ancans** [comment]
https://ngx-disqus-vcswsg.stackblitz.io

ERROR Error: StaticInjectorError[Renderer2]: 
  StaticInjectorError[Renderer2]: 
    NullInjectorError: No provider for Renderer2!

**Comment by MurhafSousli** [comment]
@kristaps-ancans I need the edit link!!

**Comment by MurhafSousli** [comment]
It seems that there is a problem with dependencies version, current workaround:
 - Downgrade to 5.0.0 and use v2.3.4
 - Upgrade to 5.0.3 and use v2.3.0

**Comment by kristaps-ancans** [comment]
Edit link - https://stackblitz.com/edit/ngx-disqus-vcswsg

Weird situation. Will try your suggestion tomorrow at work and see if it helps.

Thanks.

**Comment by MurhafSousli** [comment]
@kristaps-ancans fixed in **v2.3.6**

**Comment by kristaps-ancans** [comment]
@MurhafSousli just tested v2.3.6 - works great. Thanks.
